### DISCUSSION TITLE
How to remove /usr/local/bin/fastlane on macOS

### DISCUSSION THREAD

**Discussion by ohthepain** [fastlane/fastlane#19263] [General]
How do I get rid of  /usr/local/bin/fastlane?

I am trying to run fastlane with bundler, but I can't get rid of fastlane from /usr/local/bin

% which fastlane
/usr/local/bin/fastlane

% brew uninstall fastlane
Error: No available formula or cask with the name "fastlane". Did you mean fastme?

% gem uninstall fastlane
Gem 'fastlane' is not installed

I also tried cloning "packaged_fastlane" but there is no longer an uninstaller in there.

**Comment by lucgrabowski** [comment]
You could list files `ls -al /usr/local/bin` to see if `fastlane` is linked to another directory and if not, 
simply remove the binary: `rm /usr/local/bin/fastlane`.
Before removing the binary, you could check what is the location of the executable with `which fastlane`.
